### DISCUSSION TITLE
Realpath issues in Linux

### DISCUSSION THREAD

**Discussion by nuoyanli** [OpenModelica/OpenModelica#10967] [Q&A]
Hello developer, I encountered the following issue while executing **OMEdit on Linux**:
```
System_ Realpath failed

The named file does not exist

[/home/ly/OpenModelica/OMCompiler/Util/System. mo: 1091:5-1091:72: write] Error: Internal error System. realpath failed
```

I found relevant features in the issue: [https://github.com/OpenModelica/OpenModelica/pull/8706](https://github.com/OpenModelica/OpenModelica/pull/8706) But I don't know what caused the realpath problem. Can you prompt me to help me solve this problem

**Comment by AnHeuermann** [comment]
Transferred to issues: https://github.com/OpenModelica/OpenModelica/issues/10971
